### ISSUE TITLE
Cholera data issue - Duplicate Dates

### ISSUE THREAD

**Issue by zackarno** [OCHA-DAP/pa-global-monitoring#18]
Noticed something odd in the data. 

If we create the `df_cholera_wrangled` object using the following:

https://github.com/OCHA-DAP/pa-global-monitoring/blob/7aad68d52e44ca3f03a48340f41818ed490774c8/src/indicators/cholera/run.R#L46-L83

Then run  the code below and you will see the same date repeated w/ different start dates. 

```
df_cholera_wrangled %>% 
  group_by(iso3,date) %>% 
  filter(n()>1) %>% 
  arrange(date)
```

I haven't figured out the exact effects of this downstream, but I think it might create problems

**Comment by caldwellst** [comment]
Fixed in the `improve-comms` branch. Had noticed and tried to fix just before going on leave, but in my rush grouped by `start_date` along with `date` rather than simply grouping with `date` and taking the earliest start date. In the reports, if you go back to them, there are two separate outbreaks in the DRC being tracked, so easiest would be to simply take the total for the country and then take the earliest start date.

**Comment by zackarno** [comment]
ok great - I'll review that PR and then close the issue here

**Comment by caldwellst** [comment]
Fixed in #17.
